temporal-sdk/src/main/java/io/temporal/internal/client/external/GenericWorkflowClientImpl.java

```diff
@@ -40,7 +40,8 @@
 
 public final class GenericWorkflowClientImpl implements GenericWorkflowClient {
 
-  private static final ScheduledExecutorService executor =
+  // TODO we need to shutdown this executor
```

Comment:
This or next PR?

Reply:
another PR, at it's irrelevent to this PR. Just left a note that it should be done.